chess.py

```diff
@@ -480,9 +570,19 @@ def __init__(self, client_mode="local"):
             turnbutton.grid(row=0, column=0, sticky='nsew')
             controlbar.rowconfigure(0, weight=1)
             controlbar.columnconfigure(0, weight=1)
-            controlbar.grid(row=1, column=0, sticky='nsew')
+            controlbar.grid(row=1, column=0, columnspan=2, sticky='nsew')
```

Comment:
ik wil uiteindelijk misschien iets implementeren dat je terwijl de applicatie draait kan instellen hoe het spel eruit ziet. Ik weet niet of dat nu al zin heeft om te bepalen hoe dat gaat werken maar het is altijd handig te onthouden dat meeste onderdelen van het spel/de interface uiteindelijk optioneel/aanpasbaar gaan zijn (hopelijk)